docs/source/console.rst

```diff
@@ -22,34 +22,25 @@ Note that ``In [1]:`` and ``In [2]:`` are numbered command prompts from IPython.
 
     In [1]: from instrument.startup import *
     Activating auto-logging. Current session state plus future input saved.
-    Filename       : /path/to/project/.logs/ipython_console.log
+    Filename       : /home/prjemian/Documents/projects/prjemian/model_instrument/.logs/ipython_console.log
```

Comment:
why is this path hardcoded?

Reply:
This is output from the session.

Reply:
Requires hand-editing to restore to `path/to/project`